check_command_whitelist.rb

```diff
@@ -8,7 +8,6 @@
 require "yaml"
 
 KNOWN_REPOSITORY_PREFIXES = [
-  "git@bitbucket.org:panoramaed/",
```

Comment:
As mentioned in Slack, I think we might not need a keypair if we just added `https://github.com/panorama-ed` to this list. See https://buildkite.com/panorama-education/buildkite-assets/builds/33#d72b5999-e21e-4e8e-a61f-4d39b5e04d01

Up to you though.